setup.cfg

```diff
@@ -52,5 +52,8 @@ twine.registered_commands =
     check = twine.commands.check:main
     upload = twine.commands.upload:main
     register = twine.commands.register:main
+twine.registered_checkers =
+    longdesc = twine.commands.check:check_long_description
+    license = twine.commands.check:check_license
```

Comment:
What's the value in defining an entrypoint for `registered_checkers`? At first glance, it feels like an extra layer of complexity. Why not just call the `check_*` functions explicitly in `twine.commands.check._check_file`?

Reply:
Yes, calling the check_* functions directly would work as well. My goal here was to allow users to define project-specific checkers that may not be useful to all Python developers, or are far too strict to be included in twine itself.